### ISSUE TITLE
NetInfo is null

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#378]
In a TMPE.log:

```
[Warning] @ 1889057900 SpeedLimitManager.OnBeforeLoadData: NetInfo @ 68 is null!
   at CSUtil.Commons.Log.LogToFile(System.String log, LogLevel level)
   at CSUtil.Commons.Log.Warning(System.String s)
   at TrafficManager.Manager.Impl.SpeedLimitManager.OnBeforeLoadData()
   at TrafficManager.State.SerializableDataExtension.OnLoadData()
   at SerializableDataWrapper.OnLoadData()
   at SimulationManager.LateUpdateData(UpdateMode mode, Single minProgress, Single maxProgress)
   at SimulationManager.Managers_LateUpdateData(UpdateMode mode, Single minProgress, Single maxProgress)
   at LoadingManager+<LoadSimulationData>c__IteratorB.MoveNext()
   at AsyncTask.Execute()
   at SimulationManager.SimulationStep()
   at SimulationManager.SimulationThread()

[Warning] @ 1889086080 SpeedLimitManager.OnBeforeLoadData: NetInfo @ 69 is null!
   at CSUtil.Commons.Log.LogToFile(System.String log, LogLevel level)
   at CSUtil.Commons.Log.Warning(System.String s)
   at TrafficManager.Manager.Impl.SpeedLimitManager.OnBeforeLoadData()
   at TrafficManager.State.SerializableDataExtension.OnLoadData()
   at SerializableDataWrapper.OnLoadData()
   at SimulationManager.LateUpdateData(UpdateMode mode, Single minProgress, Single maxProgress)
   at SimulationManager.Managers_LateUpdateData(UpdateMode mode, Single minProgress, Single maxProgress)
   at LoadingManager+<LoadSimulationData>c__IteratorB.MoveNext()
   at AsyncTask.Execute()
   at SimulationManager.SimulationStep()
   at SimulationManager.SimulationThread()
```

Does this cause any problems in-game? Is there a way to track down the source of the issue?

**Comment by krzychu124** [comment]
Looks like some network prefabs are missing/broken - check output.log, there must be something

**Comment by aubergine10** [comment]
For the null netinfo message, would it be possible to output the name of non-null adjacent netinfos?

It might give some indication as to the root of the problem. For example, if it's something in NEXT or MOM mods, we'd see from name of non-null adjacent netinfos something recognisable to those mods.

**Comment by aubergine10** [comment]
Found the code, will try and output some details.

Will also try and fix #510 while in there.
